app/uk/gov/hmrc/tradergoodsprofilesdatastore/repositories/DownloadDataSummaryRepository.scala

```diff
@@ -44,14 +47,12 @@ class DownloadDataSummaryRepository @Inject() (
             .expireAfter(0, TimeUnit.SECONDS)
         ),
         IndexModel(
-          Indexes.compoundIndex(
-            Indexes.ascending("eori"),
-            Indexes.ascending("_id")
-          ),
+          Indexes.ascending("eori"),
           IndexOptions()
-            .name("eori_summaryId_idx")
+            .name("eori_idx")
```

Comment:
Nevermind, think we need to keep it - removing it causes errors such as the below. I think this is because we do use the field for searching

[info]    planner returned error :: caused by :: No indexed plans available, and running with 'notablescan'', details={}}.. (DownloadDataSummaryRepositorySpec.scala:161)
[info] - must preserve MDC *** FAILED ***
[info]   The future returned an exception of type: com.mongodb.MongoWriteException, with message: Write operation error on server localhost:27017. Write error: WriteError{code=291, message='error processing query: ns=test-DownloadDataSummaryRepositorySpec.downloadDataSummaryTree: $and
[info]       eori $eq "eori"
[info]       status $eq "FileReadyUnseen"
[info]   Sort: {}
[info]   Proj: {}
[info]   Collation: { locale: "simple" }
[info]    planner returned error :: caused by :: No indexed plans available, and running with 'notablescan'',

Reply:
coolio